app/models/track.js

```diff
@@ -16,57 +16,64 @@ const Track = Backbone.Model.extend({
 
     idAttribute:'_id',
 
-    sync: function (method, model, options) {
+    sync(method, model, options) {
         switch (method) {
             case 'create':
                 cozysdk.create('Track', model.toJSON(), (err, res) => {
                     console.log('CREATE TRACK', err, res);
+                    if (res) {
+                        model.set('_id', res._id);
+                        options.success();
+                    }
                 });
                 break;
             case 'read':
                 cozysdk.find('Track', model.get('_id'), (err, res) => {
                     console.log('READ TRACK', err, res);
+                    if (res) {
+                        options.success();
+                    }
                  });
                 break;
             case 'update':
                 cozysdk.updateAttributes(
                     'Track', model.id, model.toJSON(), (err, res) => {
+                    if (res) {
+                        options.success();
+                    }
                     console.log('UPDATE TRACK', err, res);
                 });
                 break;
             case 'delete':
                 cozysdk.destroy('Track', model.get('_id'), (err, res) => {
                     console.log('DELETE TRACK', err, res);
+                    if (res) {
+                        options.success();
+                    }
                 });
                 break;
         }
     },
 
-    updatePlays: function () {
+    getStream(callback) {
+        let ressource = this.get('ressource');
         this.set('plays', this.get('plays') +1);
         this.save();
-    },
-
-    getStreamURL: function (play) {
-        const ressource = this.get('ressource');
         switch (ressource.type) {
```

Comment:
This pattern work properly, but I suggest we keep in mind to use an `Object.assign` to add the methods to support the track type when we initialize, it'll be more flexible for next services plugins. I add a card for it in the Trello.